lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -355,29 +376,34 @@ class PointerAdapter extends BaseAdapter {
     });
   }
 
-  List<ui.PointerData> _convertEventToPointerData(
-    ui.PointerChange change,
-    html.PointerEvent evt,
-  ) {
-    final List<html.PointerEvent> allEvents = _expandEvents(evt);
+  // Transform html.PointerEvent.buttons to Flutter's PointerEvent buttons.
+  int _htmlButtonsToFlutterButtons(int buttons) {
+    // Flutter's button definition conveniently matches that of JavaScript
+    // from primary button (0x1) to forward button (0x10), which allows us to
+    // avoid transforming it bit by bit.
+    return buttons;
+  }
```

Comment:
This doesn't seem to be used anywhere, but I can't tell for sure on github :P

Reply:
Removed